src/twisted/internet/test/test_resolver.py

```diff
@@ -520,6 +520,23 @@ def test_simplifier(self):
         self.assertEqual(self.successResultOf(success), '192.168.3.4')
 
 
+    def test_portNumber(self):
+        """
+        L{SimpleResolverComplexifier} preserves the C{port} argument passed to
+        C{resolveHostName} in its returned addresses.
+        """
+        simple = SillyResolverSimple()
+        complex = SimpleResolverComplexifier(simple)
+        receiver = ResultHolder(self)
```

Comment:
It's a test helper, so it's there in case we want to add some assertions or sanity checks.